src/format/parse.rs

```diff
@@ -216,10 +215,7 @@ fn parse_rfc3339<'a>(parsed: &mut Parsed, mut s: &'a str) -> ParseResult<(&'a st
     }
 
     let offset = try_consume!(scan::timezone_offset_zulu(s, |s| scan::char(s, b':')));
-    if offset <= -86_400 || offset >= 86_400 {
```

Comment:
This check was redundant, because it is also checked in `FixedOffset_east_opt`. So I made it the same as the other places that use `Parsed::set_offset`. They can return the same `OUT_OF_RANGE`, just a little later.

Reply:
Added a comment instead.